src/types/config.ts

```diff
@@ -139,23 +139,23 @@ export type BotConfig = Static<typeof BotConfigSchema>;
 
 export const WideConfigSchema = Type.Object(
   {
-    "evm-network-id": Type.Optional(Type.Number()),
-    "price-multiplier": Type.Optional(Type.Number()),
-    "issue-creator-multiplier": Type.Optional(Type.Number()),
-    "time-labels": Type.Optional(Type.Array(LabelItemSchema)),
-    "priority-labels": Type.Optional(Type.Array(LabelItemSchema)),
-    "payment-permit-max-price": Type.Optional(Type.Number()),
-    "command-settings": Type.Optional(Type.Array(CommandItemSchema)),
-    "promotion-comment": Type.Optional(Type.String()),
-    "disable-analytics": Type.Optional(Type.Boolean()),
-    "comment-incentives": Type.Optional(Type.Boolean()),
-    "assistive-pricing": Type.Optional(Type.Boolean()),
-    "max-concurrent-assigns": Type.Optional(Type.Number()),
+    "evmNetworkId": Type.Optional(Type.Number()),
+    "priceMultiplier": Type.Optional(Type.Number()),
```

Comment:
```suggestion
    priceMultiplier: Type.Optional(Type.Number()),
```
it seems you didn't do this everywhere

Reply:
I apologize for the oversight, and I appreciate your review. I will make the necessary adjustments immediately.